spread.yaml

```diff
@@ -60,6 +60,11 @@ path: /home/gopath/src/github.com/snapcore/snapd
 exclude:
     - .git
 
+debug: |
```

Comment:
We want debug-each I think.. debug here will only work for the project-level scripts.